utils/scripts/ocm/README.md

```diff
@@ -0,0 +1,119 @@
+                        Updates the cluster information
+    install_rhods_addon
+                        Install rhods addon cluster.
+    install_gpu_addon   Install gpu addon cluster.
+    add_machine_pool    Adds machine pool to given cluster via OCM.
+    uninstall_rhods_addon
+                        Uninstall rhods addon cluster.
+    create_idp          Add an Identity providers to determine how users log into the cluster.
+```
+
+# Instructions
+Clone the [ODS-CI](https://github.com/red-hat-data-services/ods-ci.git) repository
+    
+``` git clone https://github.com/red-hat-data-services/ods-ci.git```
+    
+Navigate to ocm directory
+    
+``` cd ods-ci/utils/scripts/ocm```
+    
+### Login to OCM
+Before creating a cluster, user should login to OCM using access tokens.
+Retrieve the access token from [here](https://console.redhat.com/openshift/token/show) and login to ocm by running the below script
+    
+``` python ocm.py ocm_login --token "<token_here>"  --url=staging```
+    
+Replace the field <token_here>.
+
+*Note:* Argument --url=staging sets the target environment to deploy OSD. Mentioning _url=staging_ deploys the cluster in staging environment. If the argument removed, cluster deployed in [production](https://console.redhat.com/) environment
+
+### Create Cluster
+To deploy a OSD cluster on AWS
+
+``` python ocm.py create_cluster --aws-account-id <account_id> --aws-accesskey-id <access_key> --aws-secret-accesskey <secret_accesskey> --cluster-name <cluster_name>```
+    
+Replace the fields <account_id>, <access_key>, <secret_accesskey> and <cluster_name>.
+
+<cluster_name> accepts maximum length of 15 alpha numeric characters and word separator "-"
+
+*Note:* AWS account access_key and secret_accesskey for the user ocdCcsAdmin should be used to create a cluster, please refer [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to get more details about AWS identity and Access Management.
+
+### Install RHODS addon
+
+Once the Cluster deployed, RHODS addon can be installed using the below command
+    
+```python3 ocm.py install_rhods_addon --cluster-name <cluster_name>```
+    
+Replace the field <cluster_name> to cluster name
+
+### Create Identity Providers (IDP)
+    
+Identity providers allow us to access the cluster. Current script version supports two IDP types,
+- htpasswd
+- LDAP
+    
+##### htpasswd
+    
+```python3 ocm.py create_idp --type htpasswd --cluster <cluster_name>  --htpasswd-cluster-password <ht_password>```
+    
+Replace the fields <cluster_name> and <ht_password>
+
+The above command creates IDP type of htpasswd with the default IDP name htpasswd-cluster-admin and User name htpasswd-cluster-admin-user
+   
+##### LDAP
+
+```python3 ocm.py create_idp --type ldap --cluster <cluster_name> --ldap-bind-password <ht_password>```
+    
+Replace the fields <cluster_name> and <ht_password>
+    
+The above command creates IDP type of ldap with the password given
```

Comment:
Formatting is broken, everything is treated as a code block

Reply:
Not sure where it is broken, but the web view loads as expected.